test/unit/clarity_ext/test_units.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from clarity_ext.units import Units
+
```

Comment:
The name 'units' is slightly confusing with unit tests. maybe si_units, si_prefix?